syntax = "proto3";

package eapteka;

service RunParser {
  rpc run_parser(RequestData) returns (Product) {}
}

message RequestData {
  string city_name = 1;
  string sku = 2;
  string category = 3;
}

message Product {
  string sku = 1;
  string name = 2;
  string brand = 3;
  int32 price = 4;
  int32 price_old = 5;
  string category = 6;
}
